CHANGELOG.md

```diff
@@ -2,120 +2,112 @@
-Changelog](http://keepachangelog.com/en/1.0.0/).
+The format is based on [Keep a Changelog].
 
 > **Types of changes:**
 >
-> -   **Added**: for new features.
-> -   **Changed**: for changes in existing functionality.
-> -   **Deprecated**: for soon-to-be removed features.
-> -   **Removed**: for now removed features.
-> -   **Fixed**: for any bug fixes.
-> -   **Security**: in case of vulnerabilities.
+> - **Added**: for new features.
+> - **Changed**: for changes in existing functionality.
+> - **Deprecated**: for soon-to-be removed features.
+> - **Removed**: for now removed features.
+> - **Fixed**: for any bug fixes.
+> - **Security**: in case of vulnerabilities.
 
-[UNRELEASED](https://github.com/Qiskit/qiskit-ibmq-provider/compare/0.2.2...HEAD)
-=================================================================================
 
-Fixed
------
+## [UNRELEASED]
 
--   Fixed Qcircuit status checks according to latest API changes. (\#95)
--   Fixed an authentication issue with the `backend_status` endpoint
-    when using the new api. (\#101).
+### Fixed
 
-Changed
--------
+- Fixed Qcircuit status checks according to latest API changes. (\#95)
+- Fixed an authentication issue with the `backend_status` endpoint when using
+  the new api. (\#101)
 
--   Updated detection of classic vs. new api based on version endpoint.
-    (\#95)
--   The `requires_qe_access` decorator, previously in terra, is now
-    included in this package. (\#128)
+### Changed
 
-Added
------
+- Updated detection of classic vs. new api based on version endpoint. (\#95)
+- The `requires_qe_access` decorator, previously in terra, is now included in
+  this package. (\#128)
 
--   Added support for pooling job status via websockets. Note this is
-    only available when using the new API authentication. (\#100)
--   Added support for using object storage when submitting and
-    retrieving jobs. (\#110)
+### Added
 
-[0.2.2](https://github.com/Qiskit/qiskit-ibmq-provider/compare/0.2.1...0.2.2) - 2019-05-07
-==========================================================================================
+- Added support for pooling job status via websockets. Note this is only
+  available when using the new API authentication. (\#100)
+- Added support for using object storage when submitting and retrieving
+  jobs. (\#110)
 
-Fixed
------
 
--   Fixed Circuits parameter validation (\#89).
+## [0.2.2] - 2019-05-07
 
-[0.2.1](https://github.com/Qiskit/qiskit-ibmq-provider/compare/0.2.0...0.2.1) - 2019-05-06
-==========================================================================================
+### Fixed
 
-Fixed
------
+- Fixed Circuits parameter validation. (\#89)
 
--   Improved compatibility with older setuptools versions. (\#85)
 
-[0.2.0](https://github.com/Qiskit/qiskit-ibmq-provider/compare/0.1.1...0.2.0) - 2019-05-06
-==========================================================================================
+## [0.2.1] - 2019-05-06
 
-Added
------
+### Fixed
+.
```

Comment:
I don't think this dot is meant to be here

Reply:
Correct, thanks for spotting it!